### ISSUE TITLE
[Combobox] Toggle button should behave as toggle button

### ISSUE THREAD

**Issue by Diaan** [sl-design-system/components#3253]
### 😯 Current Behavior

Right now there is code that changes the label on the button that shows and hides the popover. In the documentation for our toggle button we [explicitly say](https://sanomalearning.design/categories/components/toggle-button/accessibility/#:~:text=If%20you%20have,panel%20is%20visible.) that changing the label on a toggle button can be confusing.

However, the code doesn't seem to work, even when the popover (listbox) is opened, it still has the label 'Show the options' 🤔

### 🤔 Expected Behavior/possible solution?

We should keep the label consistent and clear in both states. (or remove the code that isn't working as expected

### 🖥️ Steps to Reproduce

View the source when toggling the list box

### Were you able to reproduce this in our storybook?

Yes

### If yes, provide the storybook URL

https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--basic

### ⚖️ Impact

Just a visual issue / needs some polishing

### Framework

Lit

### What browsers are you seeing the problem on?

All

### Component version

0.1.7

### 👤 Your name

Diana

### 🧢 Your product/team

SLDS

**Comment by DanielleRameau** [comment]
@a11ymiko could you update this issue?

**Comment by a11ymiko** [comment]
I recommend using button that only behaves as toggle button but isn't one. 
This button should have static `aria-label='Options'` and additional `aria-expanded` that is changing as user expand/collapse the listbox.
